nautobot_golden_config/template_content.py

```diff
@@ -108,6 +112,8 @@ def right_page(self):
             .order_by("rule__feature__name")
             .values("rule__feature__name", "compliant", "non_compliant")
         )
+        if not comp_obj:
```

Comment:
making sure I understand.  If the device has no compliance object associated, return "" here basically will not render the template content at all?

Reply:
Correct